src/index.ts

```diff
@@ -445,14 +445,14 @@ function readConfig (compilerOptions: any, project: string | boolean | undefined
   delete result.config.compilerOptions.outFile
   delete result.config.compilerOptions.declarationDir
 
-  const basePath = result.path ? dirname(result.path) : cwd
+  const basePath = result.path ? normalizeSlashes(dirname(result.path)) : cwd
```

Comment:
Fixed. Interesting note is that without a `tsconfig.json` this problem did not exists for me. But as soon as i added a `tsconfig.json` i got the error. It seems that typescript handles the `basePath` differently as cwd also uses windows path separator. Either way the best would be to treat them the same.